pallets/staking/src/mock.rs

```diff
@@ -101,7 +101,7 @@ frame_support::construct_runtime!(
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
 		NodleStaking: nodle_staking::{Pallet, Call, Config<T>, Storage, Event<T>},
-		Poa: pallet_poa::{Pallet, Storage},
+		// Poa: pallet_poa::{Pallet, Storage},
```

Comment:
```suggestion
```
Use comments for comments not for versioning.

Reply:
dependency on pallate-poa is removed. now pallet staking directly refers to pallet-members